guidelines/MONAI-Workloads.md

```diff
@@ -0,0 +1,1149 @@
+
+
+
+1. Reserved capacity (always on) for QA type of use cases
+2. Shared capacity (for emergency and scheduled radiology)
+3. On-demand / R&D capacity for research workloads
+
+The estimated number of machines per modality for a large hospital will vary significantly so  instead, the number of studies per modality per month data will be used to estimate a workload description for capacity planning.  
+
+The estimated number of radiology exams per modality is provided from a few sources including data provided from three large hospitals across the US and in the UK as well as data from the UK National Health Service Diagnostic Information Dataset (DID) on RIS. 
+
+Here are the modalities derived from the data sets in descending order of average number of studies per month.
+
+
+<table>
+  <tr>
+   <td><strong>Modality</strong>
+   </td>
+   <td><strong>Other Terminology</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Plain Radiography
+   </td>
+   <td>XR, X-Ray
+   </td>
+  </tr>
+  <tr>
+   <td>Diagnostic Ultrasonography
+   </td>
+   <td>US, Ultrasound
+   </td>
+  </tr>
+  <tr>
+   <td>Computerized Axial Tomography
+   </td>
+   <td>CT, CAT
+   </td>
+  </tr>
+  <tr>
+   <td>Magnetic Resonance Imaging
+   </td>
+   <td>MR, MRI
+   </td>
+  </tr>
+  <tr>
+   <td>Fluoroscopy
+   </td>
+   <td>FL
+   </td>
+  </tr>
+  <tr>
+   <td>Nuclear Medicine Procedure
+   </td>
+   <td>NM,
+   </td>
+  </tr>
+  <tr>
+   <td>Positron Emission Tomography
+   </td>
+   <td>PET
+   </td>
+  </tr>
+</table>
+
+
+**Table:** Modalities in descending order of utilization
+
+First, for a large hospital the estimated number of radiology exams per month based on data provided by a few large hospitals.  
+
+**Figure:** Count of Imaging Activity by Monthly Modality in Large Hospitals
+
+![Large_hospitals_modality-workloads](assets/Large_hospitals_modality-workloads.png)
+
+Next, the information provided by the UK National Health Service showing the Average Count of Imaging Activity by Modality.  In this dataset, 64 of the largest institutions by volume of radiology exams were selected and averages calculated.  
+
+**Figure:** Average Count of Imaging Monthly Activity by Modality for 64 Large UK Hospitals
+
+![NHS-DID](assets/NHS-DID.png)
+
+Further data provided shows that the clinical workloads are distributed throughout the day with the peaks occurring during the middle of the day between 8AM and 6PM as would be expected aligning with normal scheduling hours.  An example of this distribution is provided in Figure: Histogram of all radiology exams per hour.
+
+
+**Figure:** Histogram of all Radiology Exams per Hour - Large Hospital
+
+![MONAI_exams_perhour_hist](assets/MONAI_exams_perhour_hist.jpg)
```

Comment:
Ideally we're being as inclusive as possible here, and that means being accessible. Being accessible means describing the contents of images and not just the names of them.